source/user-manual/manager/rbac/index.rst

```diff
@@ -51,7 +51,7 @@ Follow these steps to create an internal user, create a new role mapping, and gi
   
    #. Click **Save role mapping** to save and map the user with Wazuh as *administrator*. 
 
-   For the role mapping to take effect, make sure that ``run_as`` is set to ``true`` in the ``/usr/share/kibana/data/wazuh/config/wazuh.yml`` configuration file. Restart the Kibana service and clear your browser cache and cookies.
+   For the role mapping to take effect, make sure that ``run_as`` is set to ``true`` in the ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` configuration file. Restart the Kibana service and clear your browser cache and cookies.
```

Comment:
- Review and update instructions thoroughly for the new Wazuh components: Wazuh indexer and Wazuh dashboard. 

-  Update the screenshots to show the new Wazuh image.  

![image](https://user-images.githubusercontent.com/61882981/164481876-2f02563c-64bb-4f02-afe5-7dabb074f86b.png)